examples/mwaa/environment.yaml

```
@@ -0,0 +1,257 @@
+apiVersion: mwaa.aws.upbound.io/v1beta1
+kind: Environment
+metadata:
+  annotations:
+    meta.upbound.io/example-id: mwaa/v1beta1/environment
```

Since it took nearly 2 hours to create the resource, could you please add the following annotation?

```suggestion
    meta.upbound.io/example-id: mwaa/v1beta1/environment
    uptest.upbound.io/timeout: "7200"
```